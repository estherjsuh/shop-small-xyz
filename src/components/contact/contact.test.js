import React from 'react';
import {shallow} from 'enzyme';
import Contact from './contact.component';


describe('<Contact/>', () => {


    it('try snapshot', ()=>{
        expect(shallow(<Contact/>)).toMatchSnapshot();
    })

});