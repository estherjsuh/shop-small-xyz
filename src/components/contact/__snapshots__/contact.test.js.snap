// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact/> try snapshot 1`] = `ShallowWrapper {}`;
